src/components/tx/GasParams/index.tsx

```diff
@@ -92,11 +92,15 @@ const GasParams = ({ params, isExecution, onEdit }: GasParamsProps): ReactElemen
           </>
         )}
 
-        <Track {...MODALS_EVENTS.EDIT_ESTIMATION}>
-          <Link component="button" onClick={onEditClick} sx={{ mt: 2 }} fontSize="medium">
-            Edit
-          </Link>
-        </Track>
+        {!isExecution || (isExecution && !isLoading) ? (
```

Comment:
Why do we need to check `isExecution` here?

Reply:
If it's not an execution, the [loading flag](https://github.com/safe-global/web-core/blob/main/src/components/tx/GasParams/index.tsx#L39) doesn't matter.

Reply:
Makes sense, thanks!